src/main/java/org/broadinstitute/hellbender/engine/filters/ReadFilterLibrary.java

```diff
@@ -14,52 +12,34 @@ private ReadFilterLibrary(){ /*no instance*/ }
 
     public static final ReadFilter ALLOW_ALL_READS = read -> true;
 
-    public static final ReadFilter MAPPED =  read -> !(read.getReadUnmappedFlag() || read.getAlignmentStart() == SAMRecord.NO_ALIGNMENT_START);
-    public static final ReadFilter PRIMARY_ALIGNMENT = read -> !read.getNotPrimaryAlignmentFlag();
-    public static final ReadFilter NOT_DUPLICATE = read -> !read.getDuplicateReadFlag();
-    public static final ReadFilter PASSES_VENDOR_QUALITY_CHECK = read -> !read.getReadFailsVendorQualityCheckFlag();
+    public static final ReadFilter MAPPED =  read -> ! read.isUnmapped();
```

Comment:
having negatives in API names is bizarre. `read.isMapped()` looks so much better than `! read.isUnmapped()`